test/basicfuns.jl

```diff
@@ -44,6 +44,37 @@
     @test iszero(xlog1py(0 + im * 0, -1 + im * Inf))
 end
 
+@testset "xexpx" begin
+    @test iszero(xexpx(-Inf))
+    @test iszero(xexpx(0))
+    @test isnan(xexpx(NaN))
+
+    @test xexpx(1) ≈ exp(1)
+    @test xexpx(2) ≈ 2exp(2)
+
+    @inferred xexpx(0)
+    @inferred xexpx(0.)
+    @inferred xexpx(0f0)
```

Comment:
Can you check the type as well?

Reply:
Or maybe combine with value tests, eg
```julia
@test @inferred(xexpx(0.0)) === 0.0
```
etc.

Reply:
What do you mean?